pkg/notes/document/document.go

```diff
@@ -326,70 +322,49 @@ func sortKinds(notesByKind NotesByKind) []Kind {
 	return res
 }
 
+// urlPrefixForBucket returns the URL prefix for the provided bucket string
+func urlPrefixForBucket(bucket string) string {
+	urlPrefix := fmt.Sprintf("https://storage.googleapis.com/%s/release", bucket)
+	if bucket == "kubernetes-release" {
```

Comment:
Make this a constant in the release pkg.
ref: https://github.com/kubernetes/release/blob/b6a280c9e0c051bbe930228e807dc6bc72a32f0e/lib/releaselib.sh#L26

Reply:
Yep, will do :+1: